CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+# Guidelines for Contributing
+
+Thank you for your time on Wechaty OpenAPI.
+
+We have [gRPC](https://github.com/wechaty/grpc) for [Wechaty Puppet](https://github.com/wechaty/wechaty-puppet), and we want a RESTful API as well, so we built [OpenAPI Specification](https://www.openapis.org/) (OAS, former [Swagger](https://swagger.io/)) on top of gRPC, with the power of [gRPC OAS Gateway](https://github.com/grpc-ecosystem/grpc-gateway).
+
+## How to [contribute](https://github.com/wechaty/openapi/blob/master/CONTRIBUTING.md)
+
+- Write a blog about Wechaty OpenAPI.
+- Create a tutorial for Wechaty OpenAPI.
```

Comment:
How do you think if we add a link to [General Wechaty blog post guidelines](https://github.com/wechaty/wechaty.js.org)?

Reply:
@proudofsimin Added the link.